src/evo/specialtxman.cpp

```diff
@@ -265,3 +248,41 @@ bool UndoSpecialTxsInBlock(const CBlock& block, const CBlockIndex* pindex, llmq:
 
     return true;
 }
+
+bool CheckCreditPoolDiffForBlock(const CBlock& block, const CBlockIndex* pindex, const Consensus::Params& consensusParams,
+                                const CAmount blockReward, BlockValidationState& state)
+{
+    AssertLockHeld(cs_main);
+
+    try {
+
+        if (!llmq::utils::IsV20Active(pindex->pprev)) return true;
```

Comment:
should this check be inside try-catch?

Reply:
It was like that in `ProcessSpecialTxsInBlock` and we call this function in `ConnectBlock` like we do for `ProcessSpecialTxsInBlock` so I decided to keep that just in case.